src/egon/data/datasets/gas_grid.py

```diff
@@ -1008,6 +1009,65 @@ def insert_gas_data_eGon100RE():
     )
 
 
+def insert_gas_data_status2019():
+    """
+    Function to deal with the gas network for the status2019 scenario.
+    For this scenario just one CH4 bus is consider in the center of Germany.
+    Since OCGTs in the foreign countries are modelled as generators and not
+    as links between the gas and electricity sectors, CH4 foreign buses are
+    considered not necessary.
+
+    This function does not require any input.
+
+    Returns
+    -------
+    None.
+
+    """
+    scn_name = "status2019"
+
+    # delete old entries

```

Comment:
Does this delete data from a previous run or does it mean that there are CH4 buses created by another task that are deleted here?